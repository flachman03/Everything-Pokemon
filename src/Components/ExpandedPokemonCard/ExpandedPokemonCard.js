import React from 'react'
import { getPokemonByName } from '../../apiCalls'

export class ExpandedPokemonCard extends React.Component {
  constructor() {
    super();
    this.state = {
      pokemon: {}
    } 
  }

  componentDidMount() {
    getPokemonByName(this.props.pokemon)
      .then(data => this.setState({pokemon: data}))
  }
  render() {
    console.log(this.state.pokemon)
    return (
      <article>
        <h2>{this.state.pokemon.name}</h2>
      </article>
    )
  }
}