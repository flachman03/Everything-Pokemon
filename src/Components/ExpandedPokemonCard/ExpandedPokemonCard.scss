.expanded-move-list {
  width: 200px;
  height: 400px;
  overflow: auto;
  padding: 0;
  margin: 0 auto;
}

.expanded-move-button {
  display: block;
  width: 100%;
  margin: 2px;
  text-decoration: none;
  color: #fff;
  background: lightblue;
  border-radius: 10px;
}

.expanded-pokemon-name {
  margin-bottom: 0;
  color: #fff;
  text-decoration: underline;
}

.move-list-header {
  color: #fff;
  text-decoration: underline;
}

.expanded-poke-container {
  display: flex;
}

.display-stats-list {
  list-style-type: none;
  text-align: left;
}

.expanded-sprite-container {
  flex: 1;
  display: flex;
  flex-wrap: wrap;
}

.expanded-stat-container {
  flex: 1;
}

.expanded-move-list-container {
  flex: 1;
}

.expanded-sprite {
  height: 150px;
  width: 150px;
}

.expanded-base-stats {
  color: #fff
}

.expanded-stat-header {
  color: #fff;
  text-decoration: underline;
}