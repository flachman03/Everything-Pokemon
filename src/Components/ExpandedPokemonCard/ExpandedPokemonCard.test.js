import React from 'react'
import { shallow } from 'enzyme'
import { ExpandedPokemonCard } from './ExpandedPokemonCard'

describe('ExpandedPokemonCard', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<ExpandedPokemonCard />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})