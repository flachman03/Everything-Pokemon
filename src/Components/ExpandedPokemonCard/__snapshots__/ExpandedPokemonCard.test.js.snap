// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpandedPokemonCard should match the snapshot 1`] = `ShallowWrapper {}`;
